WordPress/Classes/ViewRelated/Me/MeViewController.m

```diff
@@ -370,20 +370,21 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                 NSString *signOutTitle = NSLocalizedString(@"Disconnecting your account will remove all of @%@’s WordPress.com data from this device.",
                                                            @"Label for disconnecting WordPress.com account. The %@ is a placeholder for the user's screen name.");
                 signOutTitle = [NSString stringWithFormat:signOutTitle, [defaultAccount username]];
-                UIActionSheet *actionSheet;
-                actionSheet = [[UIActionSheet alloc] initWithTitle:signOutTitle
-                                                          delegate:self
-                                                 cancelButtonTitle:NSLocalizedString(@"Cancel", @"")
-                                            destructiveButtonTitle:NSLocalizedString(@"Disconnect", @"Button for confirming disconnecting WordPress.com account")
-                                                 otherButtonTitles:nil];
-                actionSheet.actionSheetStyle = UIActionSheetStyleDefault;
-
-                if (IS_IPAD) {
-                    UITableViewCell *cell = [tableView cellForRowAtIndexPath:indexPath];
-                    [actionSheet showFromRect:[cell bounds] inView:cell animated:YES];
-                } else {
-                    [actionSheet showInView:self.view];
-                }
+                UIAlertController *alertController = [UIAlertController alertControllerWithTitle:signOutTitle
+                                                                                         message:nil
+                                                                                  preferredStyle:UIAlertControllerStyleActionSheet];
+                [alertController addActionWithTitle:NSLocalizedString(@"Cancel", "Cancel a prompt")
+                                              style:UIAlertActionStyleCancel
+                                            handler:nil];
+                [alertController addActionWithTitle:NSLocalizedString(@"Disconnect", @"Button for confirming disconnecting WordPress.com account")
+                                              style:UIAlertActionStyleDestructive
+                                            handler:^(UIAlertAction *alertAction) {
+                                                [self signoutFromWordPressAccount];
+                                            }];
+                UITableViewCell *cell = [tableView cellForRowAtIndexPath:indexPath];
+                alertController.popoverPresentationController.sourceRect = [cell bounds];
+                alertController.popoverPresentationController.sourceView = cell;
+                [self presentViewController:alertController animated:YES completion:nil];
```

Comment:
Just heads up. @koke recently rewrote `MeViewController` in Swift. Not sure who is gonna merge first, but his changes should get picked!.